ubuntu

```diff
@@ -7,13 +7,20 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y \
-       make git g++ gfortran ccache vim clang llvm wget sudo curl ninja-build clang-format zstd libhwloc-dev libomp-dev gnupg2 cmake libgtest-dev clang-tidy && \
+       ccache clang clang-format clang-tidy cmake curl g++ gfortran git gnupg2 libgtest-dev libhwloc-dev libomp-dev llvm make ninja-build sudo vim wget zstd && \
     if [ "$(uname -m)" = "x86_64" ]; then \
       apt-get install -y gcc-multilib g++-multilib gfortran-multilib; \
     fi && \
     apt-get purge --autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+RUN . /etc/os-release && if [ "${VERSION_ID}" != "22.04" ]; then \
```

Comment:
should the condition not read "version greater than"?
is it safe to assume that the base image is always 22.04 or greater?

Reply:
The posix `-lt` operator doesn't allow float comparison and hence this hacky solutions.

ubuntu `latest`, `rolling` and `devel` are all never that 22.04. Only reason we still have 22.04 is the intel build, hence the clearer conditional might be `[ ${INTEL} != "yes" ]`.

Reply:
I would have added a comment here to capture that we assume that `TAG` is no less than 22.04